.circleci/config.base.yml

```diff
@@ -1428,10 +1424,10 @@ commands:
                 name: Run E2e Tests
                 command: |
                   source .circleci/local_publish_helpers.sh
-                  source $BASH_ENV
                   startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
                   setNpmRegistryUrlToLocal
                   changeNpmGlobalPath
```

Comment:
why did we move this down?

Reply:
we need to include repo/node_modules/bin in the path, when spinning the image out those files take some time to appear, in order for the path to be properly updated, I needed to move it bellow this allows time for the directories to be copied